src/views/Admin.js

```diff
@@ -0,0 +1,380 @@
+import React, { useState } from 'react';
+import {
+  Layout,
+  Content,
+  Search,
+  Row,
+  Col,
+  Card,
+  Breadcrumb,
+  Header,
+  Menu,
+  Select,
+  SubMenu,
+  Affix,
+  Sider,
+  Space,
+  Tag,
+  Table,
+  Statistic,
+  Button,
+  Radio,
+  Divider,
+  Input,
+  InputNumber,
+  Popconfirm,
+  Form,
+} from '../ant';
+
+import {
+  AreaChartOutlined,
+  TeamOutlined,
+  FileProtectOutlined,
+  ArrowUpOutlined,
+  ArrowDownOutlined,
+} from '@ant-design/icons';
+
+// import antd - changed dependencies
+// import { Collapse } from 'antd';
+import { useHistory } from 'react-router';
+import Footer from '../components/Footer';
+import LoginButton from '../components/LoginButton';
+let TEMP_FRONTEND_ITEMS = [
+  { name: 'AI Design Assistant', logoURL: '/demo/aiglobal.png' },
+  { name: 'Fawkes', logoURL: '/demo/fawkes.png' },
+  { name: 'The A-Z of AI', logoURL: '/demo/theazlogo.png' },
```

Comment:
I'm not sure which one of these you are using, but go ahead and delete any unused imports here & probably TEMP_FRONTEND_ITEMS